Frontend/src/shared/services/mock/institutions.js

```diff
@@ -0,0 +1,50 @@
+const mockLeftIds = [];
+let runningId = 3;
+
+export async function createInstitution(name, description) {
+  // fake effect
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+  runningId += 1;
+  return {
+    id: runningId,
+    name,
+    country: null,
+    description,
+    profilePictureURL: null,
+    backgroundPictureURL: null,
+    adminIds: [1], // TODO: make sure that the backend puts my user ID here
+  };
+}
+
+export async function leaveInstitution(institution) {
+  mockLeftIds.push(institution.id);
+}
+
+export default async function queryMyInstitutions() {
+  return [
+    {
+      id: 1,
+      name: 'Helmholtz Institute',
+      country: 'Germany',
+      profilePictureURL: 'https://www.hzdr.de/db/Pic?pOid=55058',
+      backgroundPictureURL: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAAXNSR0IArs4c6QAAAA1JREFUGFdjsC9c9x8ABK8CXkzrf1cAAAAASUVORK5CYII=',
```

Comment:
Why not a simple url?

Reply:
I couldn't find a publically available URL for their logo, so I just encoded it with `data:image` for the mock data 😅